server/entity/lightning.go

```diff
@@ -70,8 +79,38 @@ func (li *Lightning) New(pos mgl64.Vec3) world.Entity {
 	return NewLightning(pos)
 }
 
+// WithoutDamage returns the lightning entity with damage disabled, it won't damage any entities.
+func (li *Lightning) WithoutDamage() *Lightning {
+	li.mu.Lock()
+	li.damage = false
+	li.mu.Unlock()
+	return li
+}
+
+// WithoutBlockFire returns the lightning entity but won't set any blocks on fire.
+func (li *Lightning) WithoutBlockFire() *Lightning {
+	li.mu.Lock()
+	li.blockFire = false
+	li.mu.Unlock()
+	return li
+}
+
+// WithoutEntityFire returns the lightning entity but won't set any entities on fire.
+func (li *Lightning) WithoutEntityFire() *Lightning {
```

Comment:
Does it ever make sense to not spawn fire blocks but still set entities on fire?

Reply:
I was thinking of lighting the entity on fire but not the block, or lighting none on fire